### PR TITLE
fixing provisioning dialog tests

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================

__Fixing__ scvmm/uncollecting not applicable and blocked tests.

marking and new name generator
{{pytest: cfme/tests/infrastructure/test_provisioning_dialog.py --use-provider="scvmm" --use-provider="vsphere55" --long-running  }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0aff98dbeb002de39a138b60c5d5ca611bd7dfc6:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/infrastructure/test_provisioning_dialog.py [hunk 1]
```diff
@@ -66,15 +74,14 @@ def prov_data(provisioning, provider):
...
+def provisioner(request, setup_provider, provider, vm_name):
 
     def _provisioner(template, provisioning_data, delayed=None):
         pytest.sel.force_navigate('infrastructure_provision_vms', context={
             'provider': provider,
             'template_name': template,
         })
 
-        vm_name = provisioning_data["vm_name"]
         fill(provisioning_form, provisioning_data, action=provisioning_form.submit_button)
```
Comment (psav) [inline cfme/tests/infrastructure/test_provisioning_dialog.py]: where does the vm_name get inserted into the provisioning forms now?

**Comment by dajoRH** [comment]
Lint report for commit ff30078536273f0ceb4644ad04bd5a6bce8a94c9:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 29c3fea8584e215f0ca5ee3df1c394320b35bad8:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**